webpack.config.js

```diff
@@ -1,28 +1,28 @@
 module.exports = {
```

Comment:
wait, are we actually using this file? isn't everything till running through react-scripts?

Reply:
Search me. But I'm not gonna mess with it at this point.. :-)